backend/bamboo/models.py

```diff
@@ -0,0 +1,278 @@
+from datetime import datetime
+
+from sqlalchemy import Column, Integer, String, Text, ForeignKey, \
+    Boolean, DateTime, Float
+from sqlalchemy.orm import relationship
+from werkzeug.security import generate_password_hash, check_password_hash
+
+from bamboo.core.extensions import db
+
+
+class User(db.Model):
+    __tablename__ = 'user'
```

Comment:
As you're using flask-sqlalchemy, `__tablename__` seems to be unnecessary.

Reply:
Removed.